docs/spec/index.html

```diff
@@ -1,740 +1,876 @@
+        corresponding
+        plain text <em>manifest document</em> that may be uniquely identified,
+        referenced,
+        and retrieved by a third party in a publicly accessible and documented
+        manner by the
+        UUID of the workflow definition.
       </p>
-    </section>
-
-    <section class="informative">
-      <h2>Introduction</h2>
-
       <p>
-        This specification describes how verifiable data can be shared securely
-        over an authenticated channel. Specifically, a set of OpenAPI HTTP
-        endpoints is defined, allowing authenticated data senders and receivers
-        to exchange Verifiable Presentations.
+        This workflow manifest document MUST list each required credential from
+        the
+        <a href="https://w3c-ccg.github.io/traceability-vocab/#terminology">
+          Traceability Vocabulary
+        </a>, one per line, and identified by the "type" of credential,
+        for the workflow to be considered "complete". Completeness
+        in this sense refers to an optimal state where all possible information
+        that
+        can be provided in relation to that workflow definition is present.
       </p>
-
       <p>
-        In this interaction-model, the presenting party initiates an
-        interaction, notifying the receiver. The presentation receiver responds
-        to the presentation intention with cryptographic material for completing
-        the presentation. This is used by the presenter for proving the
-        Verifiable Presentation, which finally passed to the receiver.
+        Following the credential "type" a space, followed by a numeric value
+        that ranges from
+        0 to 1 MAY be added. If this value is not provided a value of 1 MUST by
+        inferred by
+        any system using this property. This numeric value MUST be referred to
+        as "weight",
+        and may be utilized to calculate a weighted average indicating the
+        completeness of
+        a particular workflow instance.
       </p>
-
+      <!-- TODO: add example, and point to weighted avg calc -->
+      <p class="issue">
+        The standard calculation for weighted averages and algorithm for
+        calculating
+        this weighted average indicating the completeness of available
+        information
+        should be described, along with examples.
       <p>
-        The <a href="#api-spec">Open API Specification</a> section covers each
-        operation and provides example inputs and expected responses.
+      <p class="issue">
+        Stakeholders at US CBP should be consulted to determine if this weighted
+        average
+        SHOULD, MUST, or MAY be included with each traceable presentation.
+      </p>
+    </section>
+  </section>
+
+  <section class="informative">
+    <h2>Rules for Processing Data</h2>
+    <p>
+      Data exchanged according to this specification will often need to be
```

Comment:
Thanks for writing this, we've been discussing it for a while I hope this language is high level enough to support the business folks understanding the challenges of mapping graph to table data.